src/desktop/autoyast.desktop

```diff
@@ -17,6 +17,6 @@ X-SuSE-YaST-AutoInstResource=
 Icon=yast-autoyast
 Exec=/usr/bin/xdg-su -c '/sbin/yast2 autoyast'
 
-Name=Autoinstallation
```

Comment:
I'm not sure if you would need to also update this file around L58: https://svn.opensuse.org/viewvc/opensuse-i18n/trunk/lcn/50-pot/update-desktop-files-yast.pot?view=markup